lib/urbanopt/reporting/default_reports/schema/scenario_schema.json

```diff
@@ -128,6 +128,10 @@
 				"size_kw": {
 					"description": "rated power in kW",
 					"type": "string"
+				},
+				"location": {
+					"description": "Location of PV. Available options are roof and ground.",
+					"type": "string"
 				}
```

Comment:
should you make this an enum instead if the available options are roof and ground only?

Reply:
Yes, this should be an enum